scripts/get

```diff
@@ -191,9 +191,14 @@ export INPUT_ARGUMENTS="${@}"
 set -u
 while [[ $# -gt 0 ]]; do
   case $1 in
-    '--version'|-v) 
-       export DESIRED_VERSION="${2}" 
+    '--version'|-v)
        shift
+       if [[ $# -ne 0 ]]; then
+           export DESIRED_VERSION="${1}"
+       else
+           echo -e "Please provide the desired version. e.g. --verison v2.4.0 or -v latest"
+           exit 0
```

Comment:
This should exit with non-zero code

Reply:
@jascott1 , seems it non-zero code returned, fail_trap() function will be called and output will like below:
```
Please provide the desired version. e.g. --verison v2.4.0 or -v latest
Failed to install helm with the arguments provided: -v
Accepted cli arguments are:
	[--help|-h ] ->> prints this help
	[--version|-v <desired_version>] . When not defined it defaults to latest
	e.g. --version v2.4.0  or -v latest
	For support, go to https://github.com/kubernetes/helm.
```

Please help to confirm if this was the correct behavior?

Reply:
Sorry I was just thinking exit code and blanked on the trap. Lets keep it 0 and return the version required message when the flag is specified without value. Thanks @hchenxa !

Reply:
@jascott1 , any comments on message output when the desired version not provided:
```
Please provide the desired version. e.g. --version v2.4.0 or -v latest
```

Reply:
Looks great. Thanks!